lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -49,17 +52,35 @@ define(['backbone', 'jquery', 'js/common_helpers/ajax_helpers', 'js/common_helpe
                 ];
             };
 
+            createMockCohortSettings = function (isCohorted, cohortedDiscussions, alwaysCohortInlineDiscussions) {
+                return {
+                    id: 0,
+                    is_cohorted: isCohorted || false,
+                    cohorted_discussions: cohortedDiscussions || [],
+                    always_cohort_inline_discussions: alwaysCohortInlineDiscussions || true
+                };
+            };
+
+            createCohortSettings = function (isCohorted, cohortedDiscussions, alwaysCohortInlineDiscussions) {
```

Comment:
I would call this function ``createMockCohortSettings`` to mirror ``createMockContentGroups``. I would then rename the previous function as something like ``createMockCohortSettingsJson`` or ``createMockCohortSettingsOptions``.